README.md

```diff
@@ -44,27 +44,6 @@ workflows all with Python code in your job.
 
 Here is a simple example:
```

Comment:
We either need to replace this with a functioning example or remove the whole "Here is a simple example" section including the header.